src/SearchQueryService/Indexes/Models.cs

```diff
@@ -0,0 +1,101 @@
+﻿using MMLib.ToString.Abstraction;
+using System.Collections.Generic;
+
+namespace SearchQueryService.Indexes.InvoicingAzureSearch
+{
+    public class SearchIndex
+    {
+        public string Name;
```

Comment:
Toto je obyčajný field a nie property. Chýba ti `{get; set;}`.

Btw.: Tieto triedy mi prídu ako Dto-čka, ktoré by mali byť immutable, ... Podľa mňa by si na to mohol využiť [`record` type](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record). (na tie s menším počtom properties kľudne aj "positional record".